domain/src/main/java/com/zebrunner/reporting/domain/db/launcher/UserLauncherPreference.java

```diff
@@ -0,0 +1,16 @@
+package com.zebrunner.reporting.domain.db.launcher;
+
+import com.zebrunner.reporting.domain.db.AbstractEntity;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserLauncherPreference extends AbstractEntity {
```

Comment:
We used to talk about not extending abstract types designing new entities for zafira, did we?

Reply:
@tsvrko the rule is applied to spring data layout only. @nsidorevich could you resolve our dispute?